microservice/security/src/main/java/com/cumulocity/microservice/security/token/CumulocityOAuthConfiguration.java

```diff
@@ -40,6 +42,12 @@ public JwtTokenAuthenticationProvider jwtTokenAuthenticationProvider(
         return new JwtTokenAuthenticationProvider(environment, jwtAuthenticatedTokenCache);
     }
 
+    @Bean
+    @ConditionalOnMissingBean
```

Comment:
`RequestContextListener` may be redundant, so loading it only when not already in the context. Also it seems It is not always found, so initializing it here